experimental/generation/generator/templates/professional/build.cmd.lg

```diff
@@ -0,0 +1,34 @@
+# filename
+-build.cmd
```

Comment:
If you are overriding this file, you also need to override build.lg which has the bash script as well.

Reply:
Does this mean in the build.cmd.lg file, we have an if-else clause. If it has qna file, return the template with luis:build/cross-train, or else, return the template with luis:build ?

Reply:
Yes.  You can either use a function if or a template one, but the idea is the only difference is cross-train/build so capture that difference.

---
In reply to: [436673619](https://github.com/microsoft/BotBuilder-Samples/pull/2474#discussion_r436673619) [](ancestors = 436673619)